### ISSUE TITLE
New category and corresponding functionality

### ISSUE THREAD

**Issue by bwaltz** [ColbyCommunications/colby-college-theme#35]
@gregoryspragginsjr we're noticing that the left sidebar nav on the Page with Sidebar templates are not always what we expect. This is not a bug, but does occur when building out unique pages like our Course Catalogue pages.

https://master-7rqtwti-ouiqvb5juucvu.us-4.platformsh.site/academics/course-catalogue/

The catalogue needs to act a bit like a microsite of content. We have some child pages of the Catalogue like:

https://master-7rqtwti-ouiqvb5juucvu.us-4.platformsh.site/academics/course-catalogue/about-colby/
https://master-7rqtwti-ouiqvb5juucvu.us-4.platformsh.site/academics/course-catalogue/academic-program/
https://master-7rqtwti-ouiqvb5juucvu.us-4.platformsh.site/academics/course-catalogue/campus-resources/
https://master-7rqtwti-ouiqvb5juucvu.us-4.platformsh.site/academics/course-catalogue/appendices/

As you can see, the left nav is pulling in the pages under Academics because we don't have course catalogue tagged in any particular way and I think this is probably the default.

To get the child page links of the catalogue parent page to show in the left nav (emulating the department pages), could we add support for a new category called "Site?"

Basically we'd like to do the same thing we've done for the departments, where when you drill down to pages a few levels deep, the sidebar nav always stays as the immediate children of the "Department" tag. But in this case, we'd be using "Site."

I hope this makes sense. Let me know your thoughts or if you have any questions.

**Comment by gregoryspragginsjr** [comment]
@bwaltz Hey Brandon, this is another tricky one, but not too bad so no worries. As you recall, we had spoken about the course catalog and how the api response did not offer any link-offs to individual course pages. This is also reflected in the design as there were no single course page mock-ups present. So the thinking for this page was that there would be no child pages associated.

It looks like what you're trying to do with the listed pages is not necessarily provide single course pages but instead, add additional pages to provide more context for the course catalog on it's own tree.

You are correct in your assumption that we can definitely get the side-nav to auto populate like in `Departments` and `Offices` by introducing a shiny new `Course Catalog` or `Site` category if necessary in this instance. I've pushed a commit to handle just that at: https://github.com/ColbyCommunications/colby-college-theme/commit/c5ecdacd1288234d8b86ca5889d5e08d9d3d31dd

Just a few things:
- Be sure to add your `Site` category in the wordpress dashboard
- Be sure to mark your pages with this new category like what is being done on `Department`
- I suggest removing the `section-nav` component you have settled on some of these pages; that component wasn't really built for the `page_with-sidebar` template.

Let me know if this works for you.

**Comment by bwaltz** [comment]
Looks like it works the way we need.

https://master-7rqtwti-ouiqvb5juucvu.us-4.platformsh.site/academics/course-catalogue/about-colby/

Thanks!
